lib/id3c/labelmaker.py

```diff
@@ -339,6 +339,18 @@ class CollectionsUWTinySwabsLayout(LabelLayout):
     copies_per_barcode = 1
     reference = "seattleflu.org"
 
+class CollectionsUWTinySwabsHomeLayout(LabelLayout):
+    sku = "LCRY-2380"
+    barcode_type = 'UW TINY U'
+    copies_per_barcode = 1
```

Comment:
Per Zack's slack message - should this be duplicates? https://seattle-flu-study.slack.com/archives/C023B12AZCK/p1628178216034000?thread_ts=1628106734.032600&cid=C023B12AZCK

(Not sure if there was another thread, this was the most recent I'm seeing)

Reply:
Looked like it was going to be a mix for unobserved (singlets for dorms/rush, and duplicates home) so I set the default for singlets but can be overridden as needed.